source/clear-linux/guides/maintenance/validate-signatures.rst

```diff
@@ -101,7 +108,14 @@ these steps manually when performing a ``swupd update``.
 
    .. code-block:: console
 
-      openssl smime -verify -in Manifest.MoM.sig -inform der -content Manifest.MoM -CAfile Swupd_Root.pem
+      openssl smime -verify -purpose any -in Manifest.MoM.sig -inform der -content Manifest.MoM -CAfile Swupd_Root.pem
```

Comment:
@gtkramer. I receive an error message when testing the MoM. Can you please advise? 

smime: Cannot open input file Manifest.MoM, No such file or directory
smime: Use -help for summary.

We can do a quick chat if you have time.